src/components/navigation/header/config/HeaderNavigationConfig.tsx

```diff
@@ -65,8 +64,8 @@ export class HeaderNavigationConfig extends BaseConfig<HeaderNavigationConfigIns
   config: HeaderNavigationConfigInterface;
   homeUrl: string;
   menuHeight: string;
-  user: MergedUser | null;
-  userType: UserTypeExternal.Guest | MappedUserType;
+  user: EnrichedSessionUser | null;
+  userType: UserTypeExternal.Guest | Auth0UserType;
```

Comment:
Maybe for a clean up ticket, wondering if we need the guest user or that's something we could handle if user is null